main.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"os"
+
+	"github.com/google/go-github/v37/github"
+	"github.com/gorilla/mux"
+	"golang.org/x/oauth2"
+)
+
+//define global github client scoped to main package
+var client *github.Client
+
+func init() {
+	ts := oauth2.StaticTokenSource(
+		&oauth2.Token{AccessToken: os.Getenv("ACCESS_KEY")},
+	)
+	tc := oauth2.NewClient(context.Background(), ts)
+	client = github.NewClient(tc)
```

Comment:
I'm instantiating my client through the init function as I would like this function to instantiate any global singleton within my main package.